README.md

```diff
@@ -53,24 +53,139 @@ import { CalendarHeatmap } from '@manufac/reactjs-calendar-heatmap';
   color={color}
   overview={overview}
   handler={print}
+  onTooltip={show}
+  onHideTooltip={hide}
 />
 ```
 
 ### Properties
 
-| Property | Usage                                                                | Default | Required |
-| :------- | :------------------------------------------------------------------- | :-----: | :------: |
-| data     | Time series data from max a year back                                |  none   |   yes    |
-| color    | Theme hex color                                                      | #ff4500 |    no    |
-| overview | Initial overview type (choices are: year, month, day)                |  year   |    no    |
-| handler  | Handler function is fired on click of a time entry in daily overview |  none   |    no    |
+<table>
+  <tr>
+    <th>Property</th>
+    <th>Type</th>
+    <th>Usage</th>
+    <th>Default</th>
+    <th>Required</th>
+  </tr>
+  <tr> 
+    <td>
+      data
+    </td>
+    <td>
+    <!-- pre tag has been used to preserve code formatting -->
```

Comment:
@qur786 the table format in README can be removed, even if the code formatting goeas away

Reply:
@maneetgoyal sir by the above comment, I meant to say to preserve the code that we have written (data interface)

Reply:
That is okay, pre can be removed for now even if it distubrs formatting